blog/2016-12-06-HowToBecomeUSPresidentWithLessThanAQuarterOfTheVotes.adoc

```diff
@@ -33,16 +33,16 @@ Instead, let me put on my megalomaniac hat and see how I can *game the system to
 == Constraints
 
 I'll presume everyone votes, so my victory is assured regardless if they do or not.
-If no one voted, I could in theory win the elections with 11 votes: 1 vote per state in each of the 11 highest-populated states.
+If no one voted, I could in theory win the elections with 11 votes: 1 vote per state in each of the 11 most populous states.
```

Comment:
"Highest populated" sounds strange so I googled and it really seems to be used rarely. Although "most populated" wins in a google fight against "most populous", I chose the latter as it is exclusively used for example in this [Wikipedia article](https://en.wikipedia.org/wiki/List_of_U.S._states_and_territories_by_population).